test_scripts/API/VehicleData/Versioning/common.lua

```diff
@@ -0,0 +1,310 @@
+local hmiAppIds = {}
+
+local m = {}
+
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.getHMIConnection
+m.setHMIAppId = actions.setHMIAppId
+m.getHMIAppId = actions.getHMIAppId
+m.activateApp = actions.activateApp
+
+m.Step = runner.Step
+m.Title = runner.Title
+
+m.start = actions.start
+
+m.cloneTable = utils.cloneTable
+
+m.EMPTY_ARRAY = json.EMPTY_ARRAY
+m.timeout = 2000
+m.minTimeout = 500
+m.DEFAULT = "Default"
+m.vehicleData = {
+  fuelRange = {
+    value = { 
+      { 
+        type = "GASOLINE" , range = 20 
+      }, 
+      { 
+        type = "BATTERY", range = 100 
+      } 
+    },
+    type = "VEHICLEDATA_FUELRANGE"
+  },
+  tirePressure = {
+    value = {
+      pressureTelltale = "OFF",
+      leftFront = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      },
+      rightFront = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      },
+      leftRear = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      },
+      rightRear = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      },
+      innerLeftRear = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      },
+      innerRightRear = {
+        status = "NORMAL",
+        tpms = "UNKNOWN",
+        pressure = 1000
+      }
+    },
+    type = "VEHICLEDATA_TIREPRESSURE"
+  }
+}
+
+local function checkIfPTSIsSentAsBinary(bin_data)
+  if not (bin_data ~= nil and string.len(bin_data) > 0) then
+    commonFunctions:userPrint(31, "PTS was not sent to Mobile in payload of OnSystemRequest")
+  end
+end
+
+function m.getGetVehicleDataConfig()
+  return {
+    keep_context = false,
+    steal_focus = false,
+    priority = "NONE",
+    default_hmi = "NONE",
+    groups = { "Base-4", "Emergency-1", "VehicleInfo-3" }
+  }
+end
+
+local function getPTUFromPTS(tbl)
+  tbl.policy_table.consumer_friendly_messages.messages = nil
+  tbl.policy_table.device_data = nil
+  tbl.policy_table.module_meta = nil
+  tbl.policy_table.usage_and_error_counts = nil
+  tbl.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+  tbl.policy_table.module_config.preloaded_pt = nil
+  tbl.policy_table.module_config.preloaded_date = nil
+  tbl.policy_table.vehicle_data = nil
+end
+
+local function jsonFileToTable(file_name)
+  local f = io.open(file_name, "r")
+  local content = f:read("*all")
+  f:close()
+  return json.decode(content)
+end
+
+local function addParamToRPC(tbl, functional_grouping, rpc, param)
+  local is_found = false
+  local params = tbl.policy_table.functional_groupings[functional_grouping].rpcs[rpc].parameters
+  for _, value in pairs(params) do
+    if (value == param) then is_found = true end
+  end
+  if not is_found then
+    table.insert(tbl.policy_table.functional_groupings[functional_grouping].rpcs[rpc].parameters, param)
+  end
+end
+
+
+function m.preconditions()
```

Comment:
@YarikMamykin Please use `preconditions()` from module `user_modules/sequences/actions` instead of local one.